inc/saiswitch.h

```diff
@@ -2776,6 +2776,58 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_ECMP_MEMBER_COUNT,
 
+    /**
+     * @brief Random seed for adaptive routing and switching
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     */
+    SAI_SWITCH_ATTR_ARS_RANDOM_SEED,
```

Comment:
Shouldn't it be vendor-specific configuration ? Something related to your random mode ?